.claude/skills/cube-tile-tuning/hint_l1_tile.py

```
@@ -112,9 +112,10 @@
       line). For a weight-dominated decode GEMM (tiny M, huge B) DMA waste stays
       small even when MAC waste is 100%.
 
-Wall-clock proxy: tasks are dispatched in waves of `cores` (default 24).
-A 25-task config takes 2 waves on 24 cores — almost 2× slower than a
-24-task config with similar per-task work. The cost we minimize is therefore
+Wall-clock proxy: tasks are dispatched in waves of `cores` (default 24 on
+A2/A3 and 18 on A5). A 25-task A2/A3 config takes 2 waves on 24 cores—almost
+2× slower than a 24-task config with similar per-task work. The cost we
```

Ruff (0.16.0)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)